integration_tests/integration/analytics/analyticsBehaviourEntries.spec.ts

```diff
@@ -0,0 +1,227 @@
+import { getTextFromTable, testInvalidFeedbackSubmission, testValidFeedbackSubmission } from './utils'
+import Page from '../../pages/page'
+import HomePage from '../../pages/home'
+import AnalyticsBehaviourEntries from '../../pages/analyticsBehaviourEntries'
+import AnalyticsIncentiveLevels from '../../pages/analyticsIncentiveLevels'
+import GoogleAnalyticsSpy from '../../plugins/googleAnalyticsSpy'
+
+context('Analytics section', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignIn')
+    cy.task('stubAuthUser')
+    cy.task('stubCreateZendeskTicket')
+
+    cy.signIn()
+
+    const homePage = Page.verifyOnPage(HomePage)
+    homePage.viewAnalyticsLink().click()
+  })
+
+  it('has feedback banner', () => {
```

Comment:
this test is repeated identically across the 3 analytics test suites. instead i'd move it into the body of the suite after the correct nav item has been clicked. that way each page will check for the banner rather than just the incentive levels page 3 times.

Reply:
We've moved things around a bit as discussed this morning, hopefully this is similar to what you had in mind 👌 

https://github.com/ministryofjustice/hmpps-incentives-ui/pull/170/commits/3ae0a1c599122580be7152ae413d75994fff85e3